static/skin/index.js

```diff
@@ -372,6 +383,16 @@
         }
     }
 
+    function addTagElement(tagValue) {
+        const tagElement = document.getElementsByClassName('tagFilterLabel')[0];
+        tagElement.style.display = 'inline-block';
+        tagElement.innerHTML = `&#127991; ${tagValue}`;
+        tagElement.addEventListener('click', (e) => {
+            tagElement.style.display = 'none';
+            resetAndFilter('tag', '');
+        });
```

Comment:
This will keep accumulating event listeners associated with the tag label button. You should do this only once rather than on every call of `addTagElement()`